packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/footer.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Button } from '@deriv/components';
+import classNames from 'classnames';
+import { localize } from '@deriv/translations';
+import { TQuickStrategyFooter } from './components.types';
+
+const QuickStrategyFooter = ({
+    is_onscreen_keyboard_active,
+    is_mobile,
+    is_submit_enabled,
+    is_stop_button_visible,
+    setFieldValue,
+    submitForm,
+    setActiveTab,
+}: TQuickStrategyFooter) => (
+    <div
+        className={classNames('quick-strategy__form-footer', {
+            'quick-strategy__form-footer--active-keyboard': is_onscreen_keyboard_active,
+        })}
+    >
+        <Button.Group>
+            {!is_mobile && (
```

Comment:
do we need to get  `is_mobile` props from its parent since we can get it directly from '@deriv/shared';